sdk/cognitivelanguage/ai-language-text/README.md

```diff
@@ -179,635 +184,22 @@ if (result.error !== undefined) {
-      }
-      console.log(`\tClassification:`);
-      for (const classification of result.classifications) {
-        console.log(`\t\t-category: ${classification.category}`);
-      }
-    }
-  }
-}
-
-main();
-```
-
-### Action Batching
-
-Applies multiple actions on each input document in one service request.
-
-```javascript
-const { AzureKeyCredential, TextAnalysisClient } = require("@azure/ai-language-text");
-
-const client = new TextAnalysisClient("<endpoint>", new AzureKeyCredential("<API key>"));
-
-const documents = [
-  "Microsoft was founded by Bill Gates and Paul Allen.",
-  "The employee's SSN is 555-55-5555.",
-  "Easter Island, a Chilean territory, is a remote volcanic island in Polynesia.",
-  "I use Azure Functions to develop my product.",
-];
-
-async function main() {
-  const actions = [
-    {
-      kind: "EntityRecognition",
-      modelVersion: "latest",
-    },
-    {
-      kind: "PiiEntityRecognition",
-      modelVersion: "latest",
-    },
-    {
-      kind: "KeyPhraseExtraction",
-      modelVersion: "latest",
-    },
-  ];
-  const poller = await client.beginAnalyzeBatch(actions, documents, "en");
-  const actionResults = await poller.pollUntilDone();
-
-  for await (const actionResult of actionResults) {
-    if (actionResult.error) {
-      const { code, message } = actionResult.error;
-      throw new Error(`Unexpected error (${code}): ${message}`);
-    }
-    switch (actionResult.kind) {
-      case "KeyPhraseExtraction": {
-        for (const doc of actionResult.results) {
-          console.log(`- Document ${doc.id}`);
-          if (!doc.error) {
-            console.log("\tKey phrases:");
-            for (const phrase of doc.keyPhrases) {
-              console.log(`\t- ${phrase}`);
-            }
-          } else {
-            console.error("\tError:", doc.error);
-          }
-        }
-        break;
-      }
-      case "EntityRecognition": {
-        for (const doc of actionResult.results) {
-          console.log(`- Document ${doc.id}`);
-          if (!doc.error) {
-            console.log("\tEntities:");
-            for (const entity of doc.entities) {
-              console.log(`\t- Entity ${entity.text} of type ${entity.category}`);
-            }
-          } else {
-            console.error("\tError:", doc.error);
-          }
-        }
-        break;
-      }
-      case "PiiEntityRecognition": {
-        for (const doc of actionResult.results) {
-          console.log(`- Document ${doc.id}`);
-          if (!doc.error) {
-            console.log("\tPii Entities:");
-            for (const entity of doc.entities) {
-              console.log(`\t- Entity ${entity.text} of type ${entity.category}`);
-            }
-          } else {
-            console.error("\tError:", doc.error);
-          }
-        }
-        break;
-      }
-      default: {
-        throw new Error(`Unexpected action results: ${actionResult.kind}`);
-      }
-    }
-  }
-}
-
-main();
-```
+## Samples
```

Comment:
I think we can group the samples similar to OpenAI example as follows:
- Client usage: auth, modelVersion, stats, batching
- Prebuilt skills
- Custom skills
What do you think?